packages/mobile/src/screens/tip-artist-screen/SendTipScreen.tsx

```diff
@@ -90,6 +92,17 @@ export const SendTipScreen = () => {
     }
   }, [shouldFetchUserSupporter, account, receiver, dispatchWeb])
 
+  useEffect(() => {
+    if (shouldFetchSupportersForReceiver && account && receiver) {
+      dispatchWeb(
```

Comment:
What i really want, is dispatch the action within the hook so that consuming components dont have to all be touched. for now only the SendTipScreen component cares, but if we decided to directly open the tip modal for web, then the SendTip component would also need to be updated.
Want to keep trying but will timebox as this fix should be merged soon.